src/render-html/css.js

```diff
@@ -287,6 +287,10 @@ ul {
   text-align: center;
   margin: 0.5em auto;
 }
+.twitter-image img {
+  max-width: 75vw;
```

Comment:
Cool. Would you combine both `.message_inline_image img` and `.twitter-image img` with comma, so we can potentially change both.